src/data/int/modeq.lean

```diff
@@ -25,9 +25,14 @@ namespace int
 /-- `a ≡ b [ZMOD n]` when `a % n = b % n`. -/
 @[derive decidable]
 def modeq (n a b : ℤ) := a % n = b % n
-
```

Comment:
Let's leave this whitespace as it was

Reply:
https://github.com/leanprover-community/mathlib/pull/14682/commits/8c3bdf8899c83650c9ef528859a5eb11e29d0d25